site/en/docs/devtools/dom/index.md

```diff
@@ -213,6 +217,30 @@ To edit nodes as HTML with syntax highlighting and autocomplete, select **Edit a
 
    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/unvKWSLDvzoh7kHZoWbK.png", alt="Applying changes.", width="800", height="678" %}
 
+### Duplicate elements {: #duplicate }
+
+You can duplicate any element using **Duplicate element** in the node drop-down menu.
+
+{% Img src="image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/xERpA07cNN6MvlDW06ro.png", alt="Duplicate element highlighted on copy options.", width="800", height="467" %}
```

Comment:
@sofiayem I could only capture either the right-click dropdown _or_ the DevTools window when I was screenshotting so this one looks different to all the others - how do you manage to capture both using Cmd + Space + 4 + Space?

Reply:
I pasted the drop-down img to the window img ;)